client/src/pages/Actions.ts

```diff
@@ -134,6 +152,98 @@ export const getOrders = async () => {
   return (await fetchJson("GET", {}, ORDER_LIST_API, true)) || [];
 };
 
+/**
+ * Retrieves the list of items in an order.
+ *
+ * @param {string} orderName - The Spot order name in the form of {merchants/*\/orders/*}.
+ * @returns {CartItem[]} contents - The items in the order.
+ */
+export const getOrderContents = async (orderName: string) => {
+  // parse merchant ID from order name
+  let merchantId = "";
+  const matches = orderName.match(/(?<=merchants\/)(.*?)(?=\/orders)/gi);
+  if (matches) {
+    merchantId = matches[0];
+  }
+
+  // retrieve order items
+  const order = await fetchJson("GET", {}, `${ORDER_API}/${orderName}`, true);
+  let contents: CartItem[] = [];
```

Comment:
```suggestion
  const contents: CartItem[] = [];
```

Reply:
the value of contents is updated in line 181 though? `contents = order.items.map((orderItem: OrderItem) => {...});`

Reply:
oh i see! okay then!